community/neo4j-slf4j/src/test/java/org/neo4j/logging/slf4j/Slf4jLogProviderTest.java

```diff
@@ -26,6 +26,7 @@
 import org.junit.jupiter.api.Test;
 
 import java.util.ArrayList;
```

Comment:
Apologies, I think I flagged the wrong file - this was the other unused import.

Reply:
You are right, done.